en/pe-docs/docs/choreo-cli/get-started-with-the-choreo-cli.md

```diff
@@ -39,6 +40,9 @@ Run the following command to login to Choreo:
 choreo login
 ```
 
+!!! note
+    You can generate a [personal access token (PAT)](./manage-authentication-with-personal-access-tokens.md) and use it instead of interactive login. 
```

Comment:
```suggestion
    You can generate a [Personal Access Token (PAT)](./manage-authentication-with-personal-access-tokens.md) and use it instead of interactive login. 
```